File: php/delete_trunk.php
#!/usr/bin/env php
<?php

require_once '_client.php';

// Delete a trunk
$trunk_sid = getenv('TRUNK_SID');
if (!$trunk_sid) {
    echo "Error: TRUNK_SID environment variable is required\n";
    exit(1);
}

echo "Deleting trunk $trunk_sid...\n";
$result = exo_delete("/trunks?trunk_sid=$trunk_sid");
echo "Trunk deleted successfully!\n";
?> 

File: python/delete_trunk.py
#!/usr/bin/env python3

from _client import delete
import os

# Delete a trunk
trunk_sid = os.getenv('TRUNK_SID')
if not trunk_sid:
    print("Error: TRUNK_SID environment variable is required")
    exit(1)

print(f"Deleting trunk {trunk_sid}...")
result = delete(f'/trunks?trunk_sid={trunk_sid}')
print("Trunk deleted successfully!") 